src/renderers/dom/fiber/__tests__/ReactDOMFiber-test.js

```diff
@@ -44,6 +44,28 @@ describe('ReactDOMFiber', () => {
     expect(container.textContent).toEqual('10');
   });
 
+  it('should render dangerouslySetInnerHTML', () => {
```

Comment:
Let's put this test in `ReactDOMComponent-test.js` next to the other `dangrouslySetInnerHTML` tests. And give it a more specific name, like "can `dangerouslySetInnerHTML` with falsy value."